src/binding_utils.rs

```diff
@@ -171,6 +171,7 @@ macro_rules! create_exception {
                 }
             }
 
+            #[allow(dead_code)]
```

Comment:
Rework this macro to not generate this type if not required :thinking:

Reply:
Done.

Reply:
:+1: